packages/client/src/components/MyContracts.tsx

```
@@ -0,0 +1,238 @@
+
+  useEffect(() => {
+    if (!contractorId) return;
+
+    const fetchContracts = async () => {
+      setIsLoading(true);
+      setError(null);
+      
+      try {
+        const response = await getContractorAssignments(contractorId);
+        
+        if (response && response.assignments) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)